addon/TelemetryRappor.jsm

```diff
@@ -296,66 +345,91 @@ function getRandomFloat() {
  * @param {integer} cohort - Number of cohorts to use.
  * @param {string} secret - Secret to generate the Permanent Randomized Response.
  * @param {string} name - Name of the experiment.
+ * @param {constant} method  - Method used to encode. Supported: nsICryptoHash.MD5 and nsICryptoHash.SHA256
```

Comment:
nit: `method` doesn't seem like a useful name for this.

Reply:
Done. Changed to `hashMethod`.

Reply:
Done